requirements/dev.in

```diff
@@ -3,7 +3,7 @@
 -c constraints.txt
 -r pip-tools.txt          # pip-tools and its dependencies, for managing requirements files
 -r quality.txt            # Core and quality check dependencies
--r travis.txt             # tox and related dependencies
+-r ci.txt             # tox and related dependencies
```

Comment:
Nit: would be nice to keep all the comments on the right aligned on the same column.